<ui version="4.0" >
 <class>MainForm</class>
 <widget class="QDialog" name="MainForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>447</width>
    <height>307</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>ORTE demo application</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>&lt;p align="center">Publishers&lt;/p></string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="publGroup" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title" >
        <string>Colors</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="layoutWidget" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>-1</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton2_2" >
           <property name="text" >
            <string>Blue</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton2_3" >
           <property name="text" >
            <string>Green</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton2_4" >
           <property name="text" >
            <string>Red</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton1" >
           <property name="text" >
            <string>Black</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton2" >
           <property name="text" >
            <string>Yellow</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pubAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pubSource" >
       <property name="text" >
        <string>View Source</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="pixmapLabel1" >
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>32767</height>
          </size>
         </property>
         <property name="pixmap" >
          <pixmap resource="shapes.qrc" >:/ocera_logo.png</pixmap>
         </property>
         <property name="scaledContents" >
          <bool>false</bool>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1_3" >
       <property name="text" >
        <string>Don't forget to run ortemanager!</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>21</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="pixmapLabel1_2" >
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>32767</height>
          </size>
         </property>
         <property name="pixmap" >
          <pixmap resource="shapes.qrc" >:/shapes.png</pixmap>
         </property>
         <property name="scaledContents" >
          <bool>false</bool>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>22</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="About" >
         <property name="minimumSize" >
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>About</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1_2" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>&lt;p align="center">Subscribers&lt;/p></string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="subGrupColors" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>170</height>
        </size>
       </property>
       <property name="title" >
        <string>Colors</string>
       </property>
       <widget class="QWidget" name="layoutWidget_2" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>20</y>
          <width>141</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QCheckBox" name="clBlue" >
           <property name="text" >
            <string>Blue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="clGreen" >
           <property name="text" >
            <string>Green</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="clRed" >
           <property name="text" >
            <string>Red</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="clBlack" >
           <property name="text" >
            <string>Black</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="clYellow" >
           <property name="text" >
            <string>Yellow</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subSource" >
       <property name="text" >
        <string>View Source</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>pubAdd</tabstop>
  <tabstop>pubSource</tabstop>
  <tabstop>subAdd</tabstop>
  <tabstop>subSource</tabstop>
  <tabstop>About</tabstop>
 </tabstops>
 <resources>
  <include location="shapes.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>About</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subSource</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>viewSubsSource()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pubAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>addPublisher()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pubSource</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>viewPublSource()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>addSubscriber()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
